pytest/GoogleAPIAdapter_test.py

```diff
@@ -0,0 +1,18 @@
+import io
+import os
+from dynamo_backupper.GoogleAPIAdapter import GoogleAPIAdapter
+
+
+def test_upload_test_csv_file_under_folder():
+    google_api_adapter = GoogleAPIAdapter()
+
+    folder_id = google_api_adapter.find_folder_id(os.environ.get('FOLDER_NAME'))
+    assert(folder_id)
+
+    file_stream = io.StringIO()
+
+    file_stream.write('data,test,')
+
+    google_api_adapter.upload_csv_under_folder('py_test', file_stream, folder_id)
+
+    file_stream.close()
```

Comment:
Good approach but same as above; breaks 'self-contained' condition for unit tests. This seems like it would be a good use-case for Mocks. GoogleAPI seems to have a specific guide on it [here](https://github.com/googleapis/google-api-python-client/blob/c594936d82a21f68fe7a33ec3683049014084595/docs/mocks.md).

Reply:
Merging as-is on account of KT.